typings/vscode-mssql.d.ts

```diff
@@ -1235,11 +1195,7 @@ declare module "vscode-mssql" {
         getLocations(session: IAzureAccountSession): Promise<azure.subscription.Location[]>;
     }
 
-    export const enum TaskExecutionMode {
```

Comment:
removing const enums from typings files as eslint ignores those and doesn't bundle it in the out file.